sycl/test/check_device_code/matrix/matrix-nvptx-tf32-test.cpp

```diff
@@ -81,12 +81,16 @@ int main() {
           // CHECK: tail call i32 @llvm.nvvm.f2tf32.rna(float {{.*}}
           // Round a, b to tf32
           for (auto i = 0; i < 4; ++i)
-            get_wi_data(sg, sub_a)[i] =
-                round_to_tf32(get_wi_data(sg, sub_a)[i]);
+            sycl::ext::intel::experimental::matrix::get_wi_data(sg, sub_a)[i] =
+                round_to_tf32(
+                    sycl::ext::intel::experimental::matrix::get_wi_data(
+                        sg, sub_a)[i]);
 
           for (auto i = 0; i < 4; ++i)
-            get_wi_data(sg, sub_b)[i] =
-                round_to_tf32(get_wi_data(sg, sub_b)[i]);
+            sycl::ext::intel::experimental::matrix::get_wi_data(sg, sub_b)[i] =
```

Comment:
this is a cuda backend test so it should not use intel specific APIs, leave it unchanged until Jack adds the CUDA specific changes.

Reply:
Done